ui/lang/ar.js

```diff
@@ -30,20 +30,18 @@ export default {
     noData: 'لا توجد بيانات',
     noResults: 'لا توجد نتائج',
     loading: 'جارٍ التحميل...',
-    selectedRecords: function (rows) {
-      return rows === 0
+    selectedRecords: rows => (
```

Comment:
Arrow functions will break IE11. The regular function syntax is used on purpose for Qv1.

Reply:
Ok, will do, I forgot this is not transpiled